src/load.rs

```diff
@@ -0,0 +1,17 @@
+use anyhow::Result;
+use flate2::read::GzDecoder;
+use policy_evaluator::policy_fetcher::store::Store;
+use std::fs::File;
+use tar::Archive;
+
+// load policies inside the tarball provided by source_path into the default store
+pub(crate) fn load(source_path: String) -> Result<()> {
+    let default_store = Store::default();
+    let destination_path = default_store.root;
+    let tar_gz = File::open(source_path)?;
```

Comment:
I prefer to `map_err` and wrap the error into a `anyhow!("cannot open file {}: {}", source_path, e)`. This gives more clues to the user when something goes wrong.
The errors by itself is pretty dry

Reply:
make sense. done in https://github.com/kubewarden/kwctl/pull/348/commits/0798f9170407058434b66f02e205dfdf1f92ef7d